### ISSUE TITLE
[Frame] Logo for this workbench

### ISSUE THREAD

**Issue by luzpaz** [looooo/freecad.frametools#3]
Ref: https://www.freecadweb.org/wiki/Template:DevWorkbenches

**Comment by nmt** [comment]
Got any ideas of what you'd like it to look like?

**Comment by luzpaz** [comment]
Sorry for not responding. Well it should be true to the workbench name and orientation, but different than the timber workbench logo 
![image](https://user-images.githubusercontent.com/4140247/59509301-a8c4bd00-8e7e-11e9-81bb-3ceb4e247e3a.png)

**Comment by luzpaz** [comment]
@looooo need to add a logo for this WB in the Addon Manager https://github.com/FreeCAD/FreeCAD/tree/master/src/Mod/AddonManager/Resources/icons

**Comment by luzpaz** [comment]
https://github.com/FreeCAD/FreeCAD-addons/issues/37

**Comment by luzpaz** [comment]
@looooo should I open a request from the FC community to design a logo for this WB ?

**Comment by looooo** [comment]
Yes if someone finds the time to do so, go ahead. But this workbench is more a sandbox for my self for testing some things. So this is not really meant to be production-ready.

edit: I have to be more precise with spelling. This grammerly advertisment is really annoying : )
